src/reducers/real/index.js

```diff
@@ -0,0 +1,90 @@
+import {
+  ITEM_ADD,
+  ITEM_DELETE,
+  ITEM_RENAME,
+  CONTAINER_ADD,
+  CONTAINER_DELETE,
+  CONTAINER_RENAME,
+  SET_REAL_DATA
+} from 'actions/actiontypes'
+
+const initialState = {
+  _id: '',
+  name: '',
+  items: [],
+  modified_timestamp: '',
+  containers: [],
+  is_deleted: false,
+  timestamp: '',
+  snapshots: [],
+  unsnapshot_items: []
+}
+
+const realReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case ITEM_ADD:
+      return {
+        ...state,
+        items: [
+          ...state.items,
+          action.item
+        ],
+        unsnapshot_items: [
+          ...state.unsnapshot_items,
+          action.item
+        ]
+      }
+    case ITEM_DELETE:
+      let items = state.items
+      items = items.filter(ele => ele._id !== action.id)
+      return {
+        ...state,
+        items: items
+      }
+    case ITEM_RENAME:
```

Comment:
same here, let's make sure to write a test for this as well.

Reply:
What do you want to test here?

Reply:
test that the name is actually renamed properly.

Reply:
okay

Reply:
so unassigned_items should also be inside of the snapshot. 
snapshot: { container1_id: [item1_id, item2_id], 
container2_id: [item3_id]}, 
unassigned: [item4_id, item5_id, ...] 
}